-- MySQL dump 10.13  Distrib 5.5.59, for debian-linux-gnu (x86_64)
--
-- Host: localhost    Database: testCCD
-- ------------------------------------------------------
-- Server version	5.5.59-0ubuntu0.14.04.1

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `RDC`
--

DROP TABLE IF EXISTS `RDC`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `RDC` (
  `centre` varchar(100) DEFAULT NULL,
  `director` varchar(100) DEFAULT NULL,
  `adrdt` varchar(100) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=8 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `RDC`
--

LOCK TABLES `RDC` WRITE;
/*!40000 ALTER TABLE `RDC` DISABLE KEYS */;
INSERT INTO `RDC` VALUES ('Brandon','Salvano, Esther','Wang, Sen\r',1),('Morden','Salvano, Esther','Duguid, Scott \r',3),('Ottawa','Marcotte','Savard, Marc\r',4),('Saskatoon','Katepa-Mupondwa, Felicitas','Sharma, Ranjana\r',5),('Swift Current','Katepa-Mupondwa, Felicitas','McArthur, Bruce\r',6),('Lethbridge','Eudes, Francois','Plante, Yves',7);
/*!40000 ALTER TABLE `RDC` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `budget`
--

DROP TABLE IF EXISTS `budget`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `budget` (
  `year1` int(11) DEFAULT NULL,
  `year2` int(11) DEFAULT NULL,
  `category` varchar(100) DEFAULT NULL,
  `assigned_to` varchar(100) DEFAULT NULL,
  `comments` varchar(1000) DEFAULT NULL,
  `subcategory` varchar(100) DEFAULT NULL,
  `cost_type` varchar(100) DEFAULT NULL,
  `centre` varchar(100) DEFAULT NULL,
  `orig_proposal` int(11) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=58 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `budget`
--

LOCK TABLES `budget` WRITE;
/*!40000 ALTER TABLE `budget` DISABLE KEYS */;
INSERT INTO `budget` VALUES (39738,39738,'121 Miscellaneous','Administration','Variable costs (estimated at 5% per RDC, combined across centres and assigned arbitrarily to AD-RDT-Ottawa, will be split upon project approval)','Variable costs','Administrative','Ottawa',1992,1),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Badea, Ana','DNA extraction and marker analyses on barley breeding material at Brandon','Lab supplies','Direct','Brandon',1992,2),(2000,2000,'72 Materials and Supplies (Including Equipment less than $10k)','Burt, Andrew','DNA extraction and marker analyses on wheat breeding material at Brandon','Lab supplies','Direct','Brandon',1987,3),(3000,3000,'20 Travel','Cloutier, Sylvie','Expenses associated with one conference attendance or meeting for PI and PDF (3K each) for exome capture and epigenetic wheat activities led by SC.','Conferences','Direct','Ottawa',1992,4),(3000,3000,'32 Publishing, Printing, and Exposition','Cloutier, Sylvie','Expenses associated with one publication for wheat exome capture (SC) and one for epigenetics activities covered by SC (3K each).','Publications','Direct','Ottawa',1992,5),(60000,80000,'41 Professional and Special Services','Cloutier, Sylvie','The wheat exome capture activity and the wheat epigenetic exome, miRNA and RNA-Seq activities will together require 1 FTE-PDF supervised by SC and starting in late summer 2017, hence partial allocation in year 1. Budget accounts for anticipated salary and benefits at the RES01 level within the PRP program.','PRP Fellow','Direct','Ottawa',1992,6),(38500,38500,'41 Professional and Special Services','Cloutier, Sylvie','Cost of custom laboratory services for exome sample and library preparation for 192 wheat samples (averaged by year).  See detailed explanation in proposal narrative (justification of non-pay operating and service).','Service contract','Direct','Ottawa',1989,7),(45000,45000,'41 Professional and Special Services','Cloutier, Sylvie','Service expenses associated with exome activities led by SC and which include RNA QC, library preparation and QC, HiSeq sequencing for small RNA (~40K) and RNA-Seq (~50K);','Service contract','Direct','Ottawa',1992,8),(38500,38500,'72 Materials and Supplies (Including Equipment less than $10k)','Cloutier, Sylvie','Cost of laboratory supplies for exome sample and library preparation, including Roche NimbleGen kits and reagents adequate for 192 wheat samples (averaged by year). See detailed explanation in narrative (justification of non-pay operating and service).','Lab supplies','Direct','Ottawa',1989,9),(18000,18000,'72 Materials and Supplies (Including Equipment less than $10k)','Cloutier, Sylvie','Expenses associated with growing plants, tissue sampling, extraction and purification of genomic DNA and total RNA extraction, shipping, etc, for the wheat epigenetic, miRNA and RNA-Seq activities, small RNA and RNASeq led by SC.  Costs associated with exome analysis of 8 epingenomic-related samples are included in the 192 samples budgetted seperately for expome capture work.','Lab supplies','Direct','Ottawa',1992,10),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Cuthbert, Richard','DNA extraction and marker analyses on wheat breeding material at Swift Current','Lab supplies','Direct','Swift Current',1987,11),(3000,3000,'20 Travel','Diederichsen, Axel','Wheat or barley conference in 2017; American Oat Workers conference in 2018','Conferences','Direct','Saskatoon',1992,12),(0,3000,'32 Publishing, Printing, and Exposition','Diederichsen, Axel','Publications - open access fees (estimate one publication in 2018/19)','Publications','Direct','Saskatoon',1992,13),(25000,25000,'13 Students','Fedak, George','Coop students at Ottawa (2 per year) to assist with population development in wheat synthetic backcrosses','Students','Direct','Ottawa',1992,14),(4250,4250,'72 Materials and Supplies (Including Equipment less than $10k)','Fedak, George','Lab supplies for microscopy and cytogenetic analysis of wide crosses','Lab supplies','Direct','Ottawa',1992,15),(12500,12500,'13 Students','Fu, Yong-Bi','Coop student to assist with plant growth, tissue sampling, phenotypic observation, and maintenance of germplasm at PGRC','Students','Direct','Saskatoon',1989,16),(3000,3000,'20 Travel','Fu, Yong-Bi','Barley conference in 2017;  American Oat Workers Conference 2018','Conferences','Direct','Saskatoon',1989,17),(3000,3000,'32 Publishing, Printing, and Exposition','Fu, Yong-Bi','Publications - open access fees (estimate one per year)','Publications','Direct','Saskatoon',1989,18),(38500,38500,'41 Professional and Special Services','Fu, Yong-Bi','Cost of custom laboratory services for exome sample and library preparation for 192 barley samples (averaged by year).  See detailed explanation in proposal narrative (justification of non-pay operating and service).','Service contract','Direct','Saskatoon',1989,19),(38500,38500,'72 Materials and Supplies (Including Equipment less than $10k)','Fu, Yong-Bi','Cost of laboratory supplies for exome sample and library preparation, including Roche NimbleGen kits and reagents adequate for 192 barley samples (averaged by year). See detailed explanation in narrative (justification of non-pay operating and service).','Lab supplies','Direct','Saskatoon',1989,20),(0,3000,'72 Materials and Supplies (Including Equipment less than $10k)','Fu, Yong-Bi','Lab supplies for marker development and testing related to barley breeding','Lab supplies','Direct','Saskatoon',1992,21),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Graf, Rob','DNA extraction and marker analyses on wheat breeding material at Lethbridge','Lab supplies','Direct','Lethbridge',1987,22),(0,3000,'20 Travel','Hiebert, Colin','Conference travel (year 2) related to wheat molecular breeding','Conferences','Direct','Morden',1992,23),(0,3000,'32 Publishing, Printing, and Exposition','Hiebert, Colin','Open access publication (year 2) related to wheat molecular breeding','Publications','Direct','Morden',1992,24),(15000,15000,'72 Materials and Supplies (Including Equipment less than $10k)','Hiebert, Colin','Materials and supplies for marker development in wheat at Morden','Lab supplies','Direct','Morden',1987,25),(2000,2000,'72 Materials and Supplies (Including Equipment less than $10k)','Humphreys, Gavin','DNA extraction and marker analyses on wheat breeding material at Ottawa','Lab supplies','Direct','Ottawa',1987,26),(0,3000,'20 Travel','Jordan, Mark','Conference travel (year 2) related to wheat molecular breeding','Conferences','Direct','Morden',1992,27),(3000,3000,'72 Materials and Supplies (Including Equipment less than $10k)','Jordan, Mark','Materials and supplies for marker development in wheat at Morden','Lab supplies','Direct','Morden',1987,28),(1750,1750,'72 Materials and Supplies (Including Equipment less than $10k)','Knox, Ron','DNA extraction and marker analyses on wheat breeding material at Swift Current','Lab supplies','Direct','Swift Current',1987,29),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Larsen, Jamie','DNA extraction and marker analyses on wheat breeding material at Lethbridge','Lab supplies','Direct','Lethbridge',1987,30),(0,3000,'20 Travel','McCartney, Curt','Conference travel (year 2) related to wheat molecular breeding','Conferences','Direct','Morden',1992,31),(0,3000,'32 Publishing, Printing, and Exposition','McCartney, Curt','Open access publication (year 2) related to wheat molecular breeding','Publications','Direct','Morden',1992,32),(15000,15000,'72 Materials and Supplies (Including Equipment less than $10k)','McCartney, Curt','Materials and supplies for marker development in wheat at Morden','Lab supplies','Direct','Morden',1987,33),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Mitchell Fetch, Jennifer','DNA extraction and marker analyses on oat breeding material at Brandon','Lab supplies','Direct','Brandon',1989,34),(1750,1750,'72 Materials and Supplies (Including Equipment less than $10k)','Randhawa, Harpinder','DNA extraction and marker analyses on wheat breeding material at Lethbridge','Lab supplies','Direct','Lethbridge',1987,35),(12500,12500,'13 Students','Robinson, Steve','Coop students at Saskatoon to assist with laboratory work','Students','Direct','Saskatoon',1992,36),(3000,3000,'20 Travel','Robinson, Steve','Conference travel related to wheat genomics and epigenomics','Conferences','Direct','Saskatoon',1992,37),(3000,3000,'32 Publishing, Printing, and Exposition','Robinson, Steve','Publications - open access fees (estimate one per year)','Publications','Direct','Saskatoon',1992,38),(14000,0,'72 Materials and Supplies (Including Equipment less than $10k)','Robinson, Steve','Storage, workstation, and server upgrades as Saskatoon','Computers','Direct','Saskatoon',1992,39),(4800,4800,'72 Materials and Supplies (Including Equipment less than $10k)','Robinson, Steve','Funds to cover the cost of materials, reagents and safety equipment to support the work of 1 x FTE to generate:  1) 4 x DNA-seq libraries @ $120 per library 4 genotypes - Since the synthetic material contain the same alleles as their parents, only sequencing of the parental genotypes is needed. 2) 16 x WGBS libraries @ $120 per library 3) 48 x ChIP libraries @ $150 per library','Lab supplies','Direct','Saskatoon',1992,40),(1250,1250,'72 Materials and Supplies (Including Equipment less than $10k)','Ruan, Yuefeng','DNA extraction and marker analyses on wheat breeding material at Swift Current','Lab supplies','Direct','Swift Current',1987,42),(2000,2000,'72 Materials and Supplies (Including Equipment less than $10k)','Santosh, Kumar','DNA extraction and marker analyses on wheat breeding material at Brandon','Lab supplies','Direct','Brandon',1987,43),(12500,0,'13 Students','Tinker, Nick','Coop student to assist with plant growth, tissue sampling, phenotypic observation (year1)','Students','Direct','Ottawa',1989,44),(3000,3000,'20 Travel','Tinker, Nick','Year1: visit to research labs in UK, North Carolina, and/or Gatersleben who are collaborating  on design of the oat exome capture; Year 2 (July 2018): attendance at American Oat Workers conference in 2018 (NT is conference chair).','Conferences','Direct','Ottawa',1989,45),(3000,3000,'32 Publishing, Printing, and Exposition','Tinker, Nick','Publications - open access fees (estimate one per year)','Publications','Direct','Ottawa',1989,46),(45000,47000,'41 Professional and Special Services','Tinker, Nick','Partial salary and benefits for postdoctoral research fellow.  The fellow has already been staffed (Bekele) at the Res02 level, and will work slightly more than 50% of his time on designing and analysing the oat exome capture work.  He will also assist NT in overall project management and reporting.','PRP Fellow','Direct','Ottawa',1989,47),(20000,20000,'41 Professional and Special Services','Tinker, Nick','Cost of custom laboratory services for exome sample and library preparation for 48 oat samples per year.  See detailed explanation in proposal narrative (justification of non-pay operating and service).','Service contract','Direct','Ottawa',1989,48),(22500,20000,'72 Materials and Supplies (Including Equipment less than $10k)','Tinker, Nick','Cost of laboratory supplies for exome sample and library preparation, including Roche NimbleGen kits and reagents adequate for 48 oat samples per year. See detailed explanation in narrative (justification of non-pay operating and service). An additional $2500 in year 1 is for DNA preperation kits and miscellaneous lab supplies','Lab supplies','Direct','Ottawa',1989,49),(52000,52000,'13 Students','You, Frank','Coop students (4 per year) at Morden to assist with populating database, testing, and development','Students','Direct','Morden',1987,50),(6000,6000,'20 Travel','You, Frank','Conference travel related to bioinformatics (including travel for PDF)','Conferences','Direct','Morden',1987,51),(6000,6000,'32 Publishing, Printing, and Exposition','You, Frank','Bioinformatics publications - open access fees  (estimate 2 per year)','Publications','Direct','Morden',1987,52),(60000,80000,'41 Professional and Special Services','You, Frank','Research fellow in Morden - computer programming and database development. Budget accounts for hiring delay and salary increase in second year, and for anticipated sallary and benefits at the RES01 level within the PRP program.','PRP Fellow','Direct','Morden',1987,53),(57500,25500,'72 Materials and Supplies (Including Equipment less than $10k)','You, Frank','Storage and infrastructure upgrades, accounting for Shared Services procurement process,  and high-end monitors and worstations for the database development team','Computers','Direct','Morden',1987,54),(5000,5000,'72 Materials and Supplies (Including Equipment less than $10k)','You, Frank','Materials and supplies in Morden - bioinformatics lab','Lab supplies','Direct','Morden',1987,55),(0,4000,'72 Materials and Supplies (Including Equipment less than $10k)','You, Frank','Software for the Morden bioinformatics lab','Lab supplies','Direct','Morden',1987,56),(81200,75200,'72 Materials and Supplies (Including Equipment less than $10k)','Robinson, Steve','Cost of kits and reagents for sequencing to generate: 1) Skim sequencing of the four parental genotypes. Estimate 2 x Illumina lanes @ $3.5K per lane (NRC). 2) Sequencing of the WGBS libraries @ $3.5K per lane (NRC) - 16 libraries requiring 2 lanes each @$3.5K per lane. 3) Sequencing of the ChIP-seq libraries @ $3.5K per lane (NRC) – Multiplex 8 libraries require 6 lanes.','Service contract',' Direct','Saskatoon',1992,57);
/*!40000 ALTER TABLE `budget` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `collaborators`
--

DROP TABLE IF EXISTS `collaborators`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `collaborators` (
  `name1` varchar(100) DEFAULT NULL,
  `name2` varchar(100) DEFAULT NULL,
  `title` varchar(100) DEFAULT NULL,
  `role_expertise` varchar(1000) DEFAULT NULL,
  `country` varchar(100) DEFAULT NULL,
  `type` varchar(100) DEFAULT NULL,
  `affiliation` varchar(1000) DEFAULT NULL,
  `email` varchar(100) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=11 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `collaborators`
--

LOCK TABLES `collaborators` WRITE;
/*!40000 ALTER TABLE `collaborators` DISABLE KEYS */;
INSERT INTO `collaborators` VALUES ('Brian','Boyle','Chief Scientist','Head of Genome Canada funded genomics centre at Laval University. Will perform custom exome capture libraries and provide expert advice based on his experience with exome capture in large complex conifer genomes','Canada','Canadian Universities','Laval University','Brian.Boyle@ibis.ulaval.ca\r',1),('David','Konkin','Research Officer','\"Cereal genomics and computational biology. Will co-ordinating efforts with related activities of the Canadian Wheat Improvement Phase II project \"\"Genotyping Platforms Development and Support\"\"\"','Canada','Federal Departments and Agencies','National Research Council, Saskatoon.','David.Konkin@nrc-cnrc.gc.ca\r',2),('Tim','Langdon','Scientist','Access to genome sequences of diploid oat, collaboration on exome capture design','United Kingdom','Foreign Universities','Aberystwyth University; Institute of Biological, Environmental and Rural Sciences','ttl@aber.ac.uk\r',3),('Bob','Latta','Associate Professor','Specialist in evolution and adaptation of wild oat populations','Canada','Canadian universities','Dalhousie University','Robert.Latta@Dal.ca\r',4),('Martin','Mascher','Professor, head of research group','Domestication genomics in cereals: Dr Mascher has extensive experience with genome sequence analysis and the design and analysis of exon capture.  He will advise our project in strategies of exon design and data analysis.','Germany','Other Foreign Organizations','Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Biotech Campus, Gatersleben.','mascher@ipk-gatersleben.de\r',5),('Eviatar','Nevo','Professor','Plant geneticist and will collaborate on the barley germplasm selection and provide technical advice on the research','Israel','Foreign Universities','University of Haifa','nevo@research.haifa.ac.il\r',6),('Curtis','Pozniak','Professor','Wheat breeding, genome sequencing, project lead of CTAG2. Dr. Pozniak will provide us with access to the wheat genome analysis tools and sequence data to populate the database.','Canada','Canadian universities','Crop Development Centre, University of Saskatchewan','curtis.pozniak@usask.ca\r',7),('Jessica','Schlueter','Associate Professor','Access to hexaploid oat genome sequence and collaboration on exome capture design','United States','Foreign Universities','University of North Carolina, Charlotte;  College of Computing and Informatics','jschluet@uncc.edu\r',8),('Andrew','Sharpe','Senior Scientist','Sequencing, exome capture, large scale data, wheat and co-lead of CTAG2. Dr. Sharpe will provide us with access to the wheat exome and will develop the regulatory capture array that will be used to perform the bisulfite sequencing of the synthetics and their parents.','Canada','Industry associations and farming organizations','Global Institute for Food Security, Saskatoon, SK','Andrew.Sharpe@nrc-cnrc.gc.ca\r',9),('Liuling','Yan','Professor','Plant geneticist, and will collaborate on the analysis of flowering genes','United States','Foreign Universities','Oklahoma State University','liuling.yan@okstate.edu\r',10);
/*!40000 ALTER TABLE `collaborators` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `conferences`
--

DROP TABLE IF EXISTS `conferences`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `conferences` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `title` varchar(1000) DEFAULT NULL,
  `date` varchar(100) DEFAULT NULL,
  `location` varchar(100) DEFAULT NULL,
  `attendees` varchar(1000) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `conferences`
--

LOCK TABLES `conferences` WRITE;
/*!40000 ALTER TABLE `conferences` DISABLE KEYS */;
INSERT INTO `conferences` VALUES (1,'13th International Wheat Genetics Symposium','April 23-28, 2017','Tulln, Austria',' 	You, FM; McCartney; Hiebert CW; Jordan MC');
/*!40000 ALTER TABLE `conferences` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `cost__categories`
--

DROP TABLE IF EXISTS `cost__categories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `cost__categories` (
  `code` int(11) DEFAULT NULL,
  `description` varchar(100) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=14 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `cost__categories`
--

LOCK TABLES `cost__categories` WRITE;
/*!40000 ALTER TABLE `cost__categories` DISABLE KEYS */;
INSERT INTO `cost__categories` VALUES (13,'Students\r',1),(17,'Employee Benefit Plan\r',2),(20,'Travel\r',3),(32,'Publishing, Printing, and Exposition\r',4),(35,'Communications\r',5),(40,'Training\r',6),(41,'Professional and Special Services\r',7),(42,'Other Business Services\r',8),(51,'Other Rentals\r',9),(62,'Repair and Maintenance - Equipment\r',10),(72,'Materials and Supplies (Including Equipment less than $10k)\r',11),(92,'Scientific And Technical Equipment\r',12),(121,'Miscellaneous\r',13);
/*!40000 ALTER TABLE `cost__categories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `cost__subcategories`
--

DROP TABLE IF EXISTS `cost__subcategories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `cost__subcategories` (
  `description` varchar(100) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `cost__subcategories`
--

LOCK TABLES `cost__subcategories` WRITE;
/*!40000 ALTER TABLE `cost__subcategories` DISABLE KEYS */;
INSERT INTO `cost__subcategories` VALUES ('Custom Libraries\r',1),('Sequencing\r',2),('Service contract\r',3),('PRP Fellow\r',4),('Lab supplies\r',5),('Conferences\r',6),('Publications\r',7),('Computers\r',8),('Small equipment\r',9),('Students\r',10),('Variable costs\r',11);
/*!40000 ALTER TABLE `cost__subcategories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `links`
--

DROP TABLE IF EXISTS `links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `links` (
  `category` varchar(100) DEFAULT NULL,
  `subcategory` varchar(100) DEFAULT NULL,
  `software` varchar(100) DEFAULT NULL,
  `description` varchar(1000) DEFAULT NULL,
  `link` varchar(1000) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=40 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `links`
--

LOCK TABLES `links` WRITE;
/*!40000 ALTER TABLE `links` DISABLE KEYS */;
INSERT INTO `links` VALUES ('Genome annotation','Ab initio and evidence-drivable gene predictors','Augustus','Accepts expressed sequence tag (EST)-based and protein-based evidence hints. Highly accurate','http://bioinf.uni-greifswald.de/augustus/\r',1),('Genome annotation','Ab initio and evidence-drivable gene predictors','mGene','Support vector machine (SVM)-based discriminative gene predictor. Directly predicts 5\' and 3\' untranslated regions (UTRs) and poly(A) sites','http://mgene.org/\r',2),('Genome annotation','Ab initio and evidence-drivable gene predictors','SNAP','Accepts EST and protein-based evidence hints. Easily trained','http://snap.cs.berkeley.edu/\r',3),('Genome annotation','Ab initio and evidence-drivable gene predictors','FGENESH','Training files are constructed by SoftBerry and supplied to users','https://omictools.com/fgenesh-tool\r',4),('Genome annotation','Ab initio and evidence-drivable gene predictors','Geneid','First published in 1992 and revised in 2000. Accepts external hints from EST and protein-based evidence','http://genome.crg.es/software/geneid/\r',5),('Genome annotation','Ab initio and evidence-drivable gene predictors','Genemark','A self-training gene finder','http://exon.gatech.edu/GeneMark/\r',6),('Genome annotation','Ab initio and evidence-drivable gene predictors','Twinscan','Extension of the popular Genscan algorithm that can use homology between two genomes to guide gene prediction','https://omictools.com/twinscan-n-scan-tool\r',7),('Genome annotation','Ab initio and evidence-drivable gene predictors','GAZE','Highly configurable gene predictor','http://www.sanger.ac.uk/science/tools/gaze\r',8),('Genome annotation','Ab initio and evidence-drivable gene predictors','GenomeScan','Extension of the popular Genscan algorithm that can use BLASTX searches to guide gene prediction','http://genes.mit.edu/genomescan.html\r',9),('Genome annotation','Ab initio and evidence-drivable gene predictors','Conrad','Discriminative gene predictor that uses conditional random fields (CRFs)','http://www.broad.mit.edu/annotation/conrad\r',10),('Genome annotation','Ab initio and evidence-drivable gene predictors','Contrast','Discriminative gene predictor that uses both SVMs and CRFs','http://contra.stanford.edu/contrast/\r',11),('Genome annotation','Ab initio and evidence-drivable gene predictors','CRAIG','Discriminative gene predictor that uses CRFs','http://www.seas.upenn.edu/~strctlrn/craig.html\r',12),('Genome annotation','Ab initio and evidence-drivable gene predictors','Gnomon','Hidden Markov model (HMM) tool based on Genscan that uses EST and protein alignments to guide gene prediction','https://www.ncbi.nlm.nih.gov/genome/annotation_euk/gnomon/\r',13),('Genome annotation','Ab initio and evidence-drivable gene predictors','GeneSeqer','A tool for identifying potential exonâ€“intron structure in precursor mRNAs (pre-mRNAs) by splice site prediction and spliced alignment','http://www.plantgdb.org/cgi-bin/GeneSeqer/index.cgi\r',14),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','BLAT','Faster than BLAST but has fewer features','https://genome.ucsc.edu/FAQ/FAQblat.html\r',16),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Splign','Splice-aware tool designed to align cDNA to genomic sequence','https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi\r',17),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Spidey','mRNA-to-DNA alignment tool that is designed to account for possible paralogous alignments','https://www.ncbi.nlm.nih.gov/spidey/\r',18),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Prosplign','Global alignment tool that uses BLAST hits to align in a splice-site- and paralogy-aware manner','https://www.ncbi.nlm.nih.gov/sutils/static/prosplign/prosplign.html\r',19),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','sim4','Splice-aware cDNA-to-DNA alignment tool','http://pbil.univ-lyon1.fr/members/duret/cours/inserm210604/exercise4/sim4.html\r',20),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Exonerate','Splice-site-aware alignment algorithm that can align both protein and EST sequences to a genome','http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate\r',21),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Cufflinks','Extension to TopHat. Uses TopHat outputs to create transcript models','http://cole-trapnell-lab.github.io/cufflinks/\r',22),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','Trinity','High-quality de novo transcriptome assembler','https://github.com/trinityrnaseq/trinityrnaseq/wiki\r',23),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','MapSplice','Spliced aligner that does not use a model of canonical splice junction','http://www.netlab.uky.edu/p/bioinfo/MapSplice2\r',24),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','TopHat','Transcriptome aligner that aligns RNA sequencing (RNA-seq) reads to a reference genome using Bowtie to identify splice sites','https://ccb.jhu.edu/software/tophat/index.shtml\r',25),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','GSNAP','A fast short-read assembler','https://omictools.com/gsnap-tool\r',26),('Genome annotation','Choosers and combiners','JIGSAW','Combines evidence from alignment and ab initio gene prediction tools to produce a consensus gene model','http://www.cbcb.umd.edu/software/jigsaw/\r',27),('Genome annotation','Choosers and combiners','EVidenceModeler','Produces a consensus gene model by combining evidence from protein and transcript alignments together with ab initio predictions using weights for both abundance and the sources of the evidence','https://evidencemodeler.github.io/\r',28),('Genome annotation','Choosers and combiners','GLEAN','Tool for creating consensus gene lists by integrating gene evidence through latent class analysis','https://sourceforge.net/projects/glean-gene/\r',29),('Genome annotation','Choosers and combiners','Evigan','Probabilistic evidence combiner that use a Bayeisan network to weigh and integrate evidence from ab initio predictors, alignments and expression data to produce a consensus gene model','http://www.seas.upenn.edu/~strctlrn/evigan/evigan.html\r',30),('Genome annotation','Genome annotation pipelines','PASA','Annotation pipeline that aligns EST and protein sequences to the genome and produces evidence-driven consensus gene models','https://pasapipeline.github.io/\r',31),('Genome annotation','Genome annotation pipelines','MAKER','Annotation pipeline that uses BLAST and exonerate to align protein and EST sequences. Also accepts features from RNA-seq alignment tools (such as TopHat). Massively parallel','http://www.yandell-lab.org/downloads/maker/maker.tar.gz\r',32),('Genome annotation','Genome annotation pipelines','NCBI','The genome annotation pipeline from the US National Center for Biotechnology Information (NCBI). Uses BLAST alignments together with predictions from Gnomon and GenomeScan to produce gene models','https://www.ncbi.nlm.nih.gov/\r',33),('Genome annotation','Genome annotation pipelines','Ensembl','Ensembl\'s genome annotation pipeline. Uses species-specific and cross-species alignments to build gene models. Also annotates non-coding RNAs','https://www.ensembl.org/index.html\r',34),('Genome annotation','Genome browsers for curation','Artemis','Java-based genome browser for feature viewing and annotation. Can use binary alignment map (BAM) files as input','http://www.sanger.ac.uk/science/tools/artemis\r',35),('Genome annotation','Genome browsers for curation','Apollo','Java-based genome browser that allows the user to create and edit gene models and write their edits to a remote database','http://apollo.berkeleybop.org/\r',36),('Genome annotation','Genome browsers for curation','JBROWSE','JavaScript- and HTML-based genome browser that can be embedded into wikis for community work. Excellent for Web-based use','https://jbrowse.org/\r',37),('Genome annotation','Genome browsers for curation','IGV','Genome browser that supports BAM files and expression data','http://software.broadinstitute.org/software/igv/\r',38),('Genome annotation','EST, protein and RNA-seq aligners and assemblers','BLAST','Suite of rapid database search tools that uses Karlin–Altschul statistics','https://blast.ncbi.nlm.nih.gov/Blast.cgi',39);
/*!40000 ALTER TABLE `links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `milestones`
--

DROP TABLE IF EXISTS `milestones`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `milestones` (
  `milestone_id` int(11) DEFAULT NULL,
  `short_title` varchar(100) DEFAULT NULL,
  `start_date` date DEFAULT NULL,
  `end_date` date DEFAULT NULL,
  `module` varchar(100) DEFAULT NULL,
  `activity` varchar(1000) DEFAULT NULL,
  `milestone` varchar(1000) DEFAULT NULL,
  `assigned_to` varchar(1000) DEFAULT NULL,
  `progress` varchar(1000) DEFAULT NULL,
  `variance` varchar(1000) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`),
  UNIQUE KEY `milestone_id` (`milestone_id`)
) ENGINE=InnoDB AUTO_INCREMENT=72 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `milestones`
--

LOCK TABLES `milestones` WRITE;
/*!40000 ALTER TABLE `milestones` DISABLE KEYS */;
INSERT INTO `milestones` VALUES (37,'Assemble core wheat germplasm','2017-04-01','2017-05-01','Germplasm','Identify and assemble seed of 192 accessions of wheat, including eight lines relevant to in-depth epigenetic analysis: two diploid D genome parents, two tetraploid AABB genome parents and four synthetic hexaploid lines.','A core collection of 192 wheat cultivars, landraces, progenitors and synthetics of the primary gene pool, and crop wild relatives of the secondary and tertiary gene pools.','Cloutier, Sylvie; McCartney, Curt; Robinson, Steve; Diederichsen, Axel','\r',NULL,38),(38,'Assemble oat germplasm','2017-04-01','2017-05-01','Germplasm','Identify and assemble seed of 96 accessions of target germplasm in oat. Since we have recently completed and published a study of more than 300 accessions from 26 oat species (Yan et al. 2016), this germplasm is readily available, and selections will be prioritized based on this prior work.','A core collection of 96 oat cultivars, landraces, progenitors and crop wild relatives of the secondary and tertiary gene pools.','Tinker, Nicholas; Fu, Yong-Bi; Diederichsen, Axel','\r',NULL,39),(39,'Assemble barley germplasm','2017-04-01','2017-05-01','Germplasm','Identify and assemble seed of 192 accessions of target germplasm in barley. The panel will include 48 accessions representing barley cultivars, landraces, progenitors and crop wild relatives of the secondary gene pool (sampled from PGRC barley collection) as well as an application panel of 144 lines of Israeli wild barley (H. spontaneum) sampled from 200 available lines with variable flowering time provided by Prof. Eviatar Nevo.','A core collection of 192 barley cultivars, landraces, progenitors and crop wild relatives of the secondary gene pool, including 144 accessions of wild barley sampled from natural populations over time.','Fu, Yong-Bi; Diederichsen, Axel; Badea, Ana','\r',NULL,40),(40,'Design first iteration of gene capture in oat.','2017-05-01','2017-07-01','Exome','Design the first iteration of a novel gene capture array for oat using Roche NimbleGen SeqCap Target Enrichment (EZ System).  Roche will provide the oligo design based on our prediction of coding sequences, including coding regions and intron-exon junctions. Our design will be based on a first round of gene calls predicted from either or both of a Pac-Bio based sequence (Schlueter, NSF) from \'GS7\' oat, or an Illumina/NRGene sequence of \'Firth\' expected to be available from an ad-hoc consortium with collaborators by the start of this project. If a hexaploid reference and gene predictions are not available, the first exome will be designed from available cDNA sequences guided by a diploid oat sequence available from T. Langdon.  Predictions will include all matches to the CORE RNAseq assemblies from 4 oat tissues (Tinker et al 2016), all other public cDNA predictions, and additional de-novo predictions from the reference sequence. Estimated design space will be 60 MBP while capture space','First round gene-capture design for oat, annotated and compared to existing wheat and barley gene capture assays.','Tinker, Nicholas;','\r',NULL,44),(41,'Obtain wheat and barley exome capture assays.','2017-04-01','2017-06-01','Exome','Obtain/evaluate Roche NimbleGen SeqCap kits for the most recent public gene capture arrays for wheat and barley.  These are public and no cost is incurred other than for the kits and reagents. Unless further refinements are made prior to selection, the barley exome assay will be 120426_Barley_BEC_D04.EZ (89 Mb) and wheat will be 120426_Wheat_WEC_D02 (107 Mb).','Gene capture strategies for wheat and barley based on commercial designs.','Cloutier, Sylvie; Fu, Yong-Bi','\r',NULL,45),(42,'Grow germplasm and isolate DNA','2017-05-01','2017-07-01','Germplasm','Grow target germplasm from wheat, oat, and barley as single plants from each accession. Extract DNA by non-destructive harvesting of leaf samples from juvenile plants, and allow plants to mature for next milestone (seeds). Seeds and DNA from single plants of most oat accessions are already available, but selected plants will be regrown or added. Some oat and wheat lines will require vernalization, and these lines will be started early and vernalized in anticipation of this project.','High quality DNA from 480 core wheat, barley, and oat accessions.','Cloutier, Sylvie; Tinker, Nicholas; Fu, Yong-Bi','\r',NULL,46),(44,'Analyze SNPS in existing wheat sequence','2018-01-01','2018-03-31','Database','Using the developed SNP mining tool and the latest wheat reference (IWGSC Chinese Spring v1.0 or later) mine and analyze SNPs in collection of exome sequences from more than 187 wheat lines, available from prior research and from publicly available exome sequence data (NCBI SRA, the CTAG2 project), and populate wheat database.','An array of high-quality wheat genome-specific SNPs from the currently available wheat genomes will be accessible in the database for marker development and breeding use.','You, Frank; McCartney, Curt','\r',NULL,48),(45,'Observe plants and produce seed','2017-07-01','2018-04-01','Germplasm','Continue to grow all single plants (from DNA step) from wheat, oat, and barley to maturity. Take notes and photographs of plants, and obtain seeds. Progeny of the initial single plants may be produced afterward for additional increase and for submission as type specimens to the Canadian vascular herbarium.','Single-plant-derived seed, phenotype notes, and photographs from 480 core wheat, barley, and oat accessions. Herbarium type specimens will follow.','Cloutier, Sylvie; Tinker, Nicholas; Fu, Yong-Bi','\r',NULL,49),(46,'Exome capture and sequencing (first round)','2017-07-01','2017-11-01','Exome','Produce gene capture libraries for 48 samples in each of oat, wheat, and barley (first round samples). Estimate library quality and proceed with next generation (HiSeq 4000) sequencing at a coverage of ~25 to 50 M reads per genotype.  The first round of fragmentation, barcoding, library preparation, and exome capture in all three crops using the best service provider.','Raw sequence at a coverage of ~25 to 50 M reads per genotype at an estimated target depth of 50x will be generated.','Cloutier, Sylvie; Tinker, Nicholas; Fu, Yong-Bi','\r',NULL,50),(47,'Test SNP database in wheat','2017-08-01','2019-03-31','Breeding','Since the wheat database will be populated first from existing exome data, the first tests will be performed here. SNP loci will be retrieved from the SNP database to be validated as KASP assays based upon location of a SNP in the genome and predicted polymorphism in parents of bi-parental crosses. KASP assays will be tested in existing mapping populations segregating for genes of interest (e.g. wheat midge resistance gene Sm1, Ug99 stem rust resistance gene SrCad, leaf rust resistance gene Lr16, and pre-harvest sprouting resistance QTL). This activity is expected to be iterative over the course of the project following initial development of the database.','SNP loci placed on existing linkage maps. Validation that the SNP calling pipeline is functioning properly.','McCartney, Curt; Hiebert, Colin; Jordan, Mark; You, Frank','\r',NULL,51),(48,'RNA sequencing','2017-09-01','2018-09-01','Epigenetics','High quality total RNA from the genotypes and tissues described in milestone 40 will be QCed, m RNA libraries will be constructed and sequenced on the best available chemistry at the time of sequencing (currently using HiSeq4000 producing 2X 300MB paired-end tag reads of 100bp) to produce ~25M reads per genotype/tissue/replicate. This will be performed as a service contract using the best service provider. ','Transcriptome sequence of the target germplasm for numerous tissues and conditions of the progenitors and synthetics with a throughput of ~25M reads per sample.','Cloutier, Sylvie','\r',NULL,52),(49,'Small RNA sequencing','2017-09-01','2018-09-01','Epigenetics','High quality total RNA from 3 biological replicates of pre- and post-vernalization meristems of the 4 progenitors and 4 synthetics and of ovary and anther tissues will be QCed, small RNA libraries will be constructed, QCed and sequenced using the best available chemistry at the time of sequencing for this type of libraries (currently using HiSeq 50bp single-end read with a throughput of 150M reads per lane) to produce ~25M reads per genotype/tissue/replicate. This will be performed as a service contract using the best service provider.','Small RNA sequence of the target germplasm for numerous tissues and conditions of the progenitors and synthetics with a throughput of ~25M reads per sample.','Cloutier, Sylvie','\r',NULL,53),(50,'Genome-wide survey sequencing','2017-04-01','2018-03-31','Epigenetics','Sequence will be generated from targeted regions of the genome in each of the 4 progenitor genotypes described in milestone 40. Targeted genomic regions generated through hybridization of DNA-seq libraries to the exome (available) and regulatory-region capture arrays (CTAG2). Alignment to the CS reference genome will identify relevant SNP that can confound WGBS data.  Read alignments will use Bowtie2 and custom Perl scripts to interpret the SNP data.','The skim sequence generated over the targeted regions will be used to identify underlying SNP polymorphisms between the progenitor lines and their homologous regions in the CS reference sequence.','Robinson, Steve','\r',NULL,54),(51,'bisulfite sequencing ','2017-04-01','2018-09-30','Epigenetics','Whole genome bisulfite sequencing (WGBS) libraries will be generated and hybridized to regulatory region capture arrays designed to capture bisulfite converted and non-converted bases (CTAG2).  Leaf tissue (two replicates) will be sampled from the eight genotypes (milestone 40) both pre- and post- vernalization.  The sequences will be aligned to the CS genome where SNP variation identified in milestone 75 has been incorporated.  Read alignment will be performed using Bismark and established analysis pipelines will be used to analyze WGBS data (CTAG2). ','DNA methylation profiles targeted to the regulatory regions of the annotated genes will be generated from leaf material.  These data will provide a layer of epigenetic information that can be integrated with the data generated in milestones 65, 70, 85.','Robinson, Steve','\r',NULL,55),(52,'Histone modification analysis','2017-09-01','2018-09-30','Epigenetics','Chromatin immunoprecipitation (ChIP) will be used to purify histones the following epigenetic marks H3K4ac, H3K4me3, H3K27ac and H3K27me3 along with IgG controls from leaf tissue both pre- and post- vernalization.  A total of three replicates will be used to assess the variability between the samples.  ChIP-seq reads will be aligned to the reference genome and peaks called using the MACS2 software. ','Histone modifications targeting H3K4ac, H3K4me3, H3K27ac and H3K27me3 will be mapped throughout the genome generated from leaf material.  These data will an additional layer of epigenetic information that can be integrated with the data generated in milestones 65, 70, 80.','Robinson, Steve','\r',NULL,56),(53,'Assemble gene space (first round). ','2017-11-01','2018-02-01','Exome','Assemble raw exome captured sequence into gene space relative to the latest reference genomes. Reads from cultivated accessions will be assembled directly to the reference genomes (selected in the database design) using the BWA short-read assembler, producing a mapping file in BAM format, from which coverage estimation and SNP calling will be performed using the developed allele mining tool. For the non-cultivated species, the gene space will be assembled de novo using CLC Assembly Cell (http://www.clcbio.com) since we expect gene space and captured introns to deviate more substantially from the design space. Following this, two approaches will be taken. First, the assembled consensus reads from each accession will be aligned to the most similar location in the cultivated reference genome with a threshold matching criteria, then remaining sequences that will be used to identify potential genes that are novel or not represented in the cultivated genome. Such targets may be captured beca','Assembled gene space relative to cultivated genomes in wheat, oat, and barley.','Tinker, Nicholas; Fu, Yong-Bi; You, Frank','\r',NULL,57),(54,'Test new SNPs in wheat, barley, and oat (first round exomes)','2018-02-01','2019-03-31','Breeding','Testing and implementation of SNPs from the first new exome assemblies using the SNP calling pipeline in Canadian wheat, oat, and barley breeding programs. Expected targeted genes in wheat are Sm1, SrCad, Lr16, and the pre-harvest sprouting resistance QTL located on chromosome 4A. In oat, new assays for Pc91 and Pc94 may be developed, and targets for barley will be selected at a later date. ','SNP data on current Canadian wheat, oat and barley breeding germplasm, including different wheat marketing classes from across the country. Validation that the SNP assays are predictive of the presence of targeted genes. ','McCartney, Curt; Hiebert, Colin; Jordan, Mark; Tinker, N; Mitchel Fetch, J. Fu, Y-B.; Badea, A. You,','\r',NULL,58),(56,'Perform in-depth sequence analysis on candidate genes','2018-02-01','2019-03-31','Exome','Project participants will perform in-depth sequence analysis on candidate genes that show homology to genes implicated in fungal resistance (RGA), flowering time, cellulose synthesis, and hull adhesion. This work will commence as soon as analysis of the first round of exome capture is complete, and continue to the end of the project, with ongoing recommendations for continued work.','Discovery of new alleles for genes that are not present in primary gene pools of each species. Recommendations for further work and validation experiments using diverse alleles from target traits. ','Cloutier, Sylvie; Tinker, Nicholas; Fu, Yong-Bi; You, Frank','\r',NULL,59),(57,'Phylogeny analysis','2018-02-01','2019-03-31','Exome','Project participants will perform additional phylogeny analysis using defined gene sets at a level of species-diversity and gene coverage never-before achieved in these crops.  This work will be performed primarily in oat, where significant uncertainty remains regarding species divergence and common ancestors. The oat work will be supplemented by new chloroplast sequencing being conducted by our collaborator (Robert Latta). Other collaborations may be sought to examine phylogeny analysis in oat and barley. This work will commence as soon as analysis of the first round of exome capture is complete, and continue to the end of the project, with ongoing recommendations for continued work.','Refined hypotheses regarding phylogeny of crop wild ancestors.  This information will be used to develop new germplasm conservation efforts, and to make decisions regarding the implementation of wide crosses.','Fu, Yong-Bi; Tinker, Nicholas; Cloutier, Sylvie;','\r',NULL,60),(58,'Add first-round exomes to database','2018-02-01','2018-03-01','Database','Re-populate databases to capture additional SNP data from the first-round exome sequencing, de novo assembly and allele mining. Tracks will be added to JBrowse  to visually display alignments of gene copies, de novo assembled gene sequences and alleles for each of accessions','A database with all data generated from the first round exome sequencing and assembly. The data would be viewable through JBrowse and the alleles from the round of accessions for three crops will be searchable through web search engine.  ','You, Frank; Tinker, Nick; Fu, Yong-Bi; Cloutier, Sylvie','\r',NULL,61),(59,'Revise Oat exome capture assay','2018-02-01','2018-04-01','Exome','Evaluate quality of the oat exome capture and revise the design. Since the first exome capture results will provide data on read coverage in cultivated materials, revisions can be made to eliminate or re-design targets with poor coverage and eliminating oligos or entire targets that match repetitive reads (reads match multiple targets). This will improve the efficiency of the next design and increase the depth of sequencing. The new design may also add new sequences from refined gene predictions.','A revised oat exome capture design','Tinker, Nicholas;','\r',NULL,62),(60,'Second round of gene capture and sequencing','2018-04-01','2018-07-01','Exome','Perform the second round of gene capture assays and sequencing in 144 samples of each of wheat and barley, and 48 samples of oat using the next generation of the oat exome design, and accounting for any revisions suggested by the first round (e.g. protocols or sequencing depth). In barley, the second round will include the 144 wild barley lines to characterize early flowering time genes. The second round of exome capture in wheat and barley could be performed earlier relative to oat since no assay revision is required. The timing of this work allows the costs for the second round of exomes in wheat and barley to be adjusted between years, as determined by the budget status at the time.','Raw sequence and assembled data for additional 144 lines in each of barley and wheat, and 48 additional lines in oat.','Cloutier, Sylvie; Tinker, Nicholas; Fu, Yong-Bi;','\r',NULL,63),(61,'Second round of exome assembly and analysis','2018-07-01','2018-10-01','Exome','Perform second round of mapping, de novo assembly and analysis of exome capture reads, as before, incorporating any new lessons or experience. The second round of exome analysis in wheat and barley could be advanced relative to oat since no assay revision is required.','Assembled gene space relative to cultivated genomes in wheat, oat, and barley in additional germplasm.','Tinker, Nicholas; Cloutier, Sylvie; Fu, Yong-Bi; You, Frank','\r',NULL,64),(62,'Epigenetic database layers in wheat','2018-10-01','2019-03-31','Database','Integrate RNA sequence, small RNA sequences, genome-wide survey sequence and bisulfate sequences to the GBrowse database using GFF formatted files accessible through the modified JBrowse interface. ','A wheat epigenetic database will be searchable. ','You, Frank; Cloutier, Sylvie; Robinson, Steve','\r',NULL,65),(63,'Add second-round exomes to database','2018-10-01','2019-03-31','Database','Re-populate databases to capture additional sequence and SNP data from newly sequenced accessions, and add additional tracks to the databases to show data for all accessions for each crop. At this stage, all data, including sequences, alleles and annotation results for exome capture and epigenetic analysis will be stored into the databases and are accessible from the web application. The database and web application will be further tested and optimized. ','Databases and their web applications with all data generated in this project and complete search functions will be deployed to AAFC scientists and collaborators. All raw data and annotated data will be downloadable.','You, Frank; Tinker, Nick; Fu, Yong-Bi','\r',NULL,66),(34,'Develop allele mining tool','2017-04-01','2017-12-31','Database','Develop an allele mining tool: Based on the existing SNP mining pipeline “AGSNP” developed by Frank You (You et al. 2011, 2012 and Kumar et al. 2012), a new allele mining platform suitable for both diploid and polyploid species will be optimized. This pipeline basically adopts BWA and SAMtools for read mapping and variant calling. A dedicated algorithm for allele detection in polyploid and quality control will be developed, optimized and validated using empirical SNP assays. Perl and C++ will be used to develop the pipeline tool.','An allele mining package optimized for cereal crops ','You, Frank;',' ',' ',67),(35,'Database design','2017-04-01','2017-08-31','Database','Design initial databases and web applications. The open GMOD database will be adopted in the database design. The databases will be based on current public reference genome sequences from hexaploid wheat (cv ‘Chinese Spring’), barley (cv ‘Morex’) and oat (cv ‘GS7’). JBrowse will be used to browse all alleles of each genotype or accessions for each species of wheat, barley and oat. A search engine will be designed to allow users to conveniently search for a specific gene, a genomic region, a sequence or a pair of marker sequences. Primer design tools will be also integrated to automate the marker development for downstream genotyping, such as KASPER assay. ','A database framework with preliminary implementation of the database and web applications for further implementation.','You, Frank; McCartney, Curt',' ',' ',68),(36,'Develop database web application ','2017-04-01','2018-12-31','Database','Develop a web application for the database system for data management and user queries. The diversity component of the FlaxDB (under development by Dr. Frank You’s team for a WGRF project) will be adopted for further improvement as an allele search engine. JBrowse will be further improved or configured for displaying tracks of large numbers of genotypes. The popular online primer design program “BatchPrimer3” developed by Dr. Frank You (You et al. 2008) will be improved and integrated to design KASPER primers for users’ SNP search results. Online runtime statistical analysis and annotation tools for diversity will be developed by calling R functions. The database and applications will be further optimized after all data has been populated.','A user-friendly cereal allele database and its web applications with allele data from three cereal crops which are directly accessible within AAFC intranet by AAFC scientists and by collaborators in this project.','You, Frank; McCartney, Curt',' ',' ',69),(43,'Isolate RNA','2017-05-01','2018-04-01','Epigenetics','Grow single seed-derived progeny of 4 progenitors and 4 synthetic lines, sample meristem tissues pre- and post-vernalization, extract total RNA and perform quality control. This experiment will include three biological replicates. From the same 8 genotypes, 16 tissues (hypocotyl, root, leaf, anther, ovaries, early developing seeds at different stages, stem, etc.) will be similarly processed.','High quality RNA will be generated for the construction of small RNA and mRNA libraries for deep coverage sequencing.','Cloutier, Sylvie',' ',' ',71);
/*!40000 ALTER TABLE `milestones` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `news`
--

DROP TABLE IF EXISTS `news`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `news` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `date` datetime DEFAULT NULL,
  `entry` varchar(1000) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `news`
--

LOCK TABLES `news` WRITE;
/*!40000 ALTER TABLE `news` DISABLE KEYS */;
/*!40000 ALTER TABLE `news` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `objectives`
--

DROP TABLE IF EXISTS `objectives`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `objectives` (
  `order` int(11) DEFAULT NULL,
  `short` varchar(100) DEFAULT NULL,
  `description` varchar(1000) DEFAULT NULL,
  `outcome` varchar(1000) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`),
  UNIQUE KEY `order` (`order`)
) ENGINE=InnoDB AUTO_INCREMENT=29 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `objectives`
--

LOCK TABLES `objectives` WRITE;
/*!40000 ALTER TABLE `objectives` DISABLE KEYS */;
INSERT INTO `objectives` VALUES (20,'SNP pipeline','A high-throughput and efficient bioinformatics pipeline for systematic variant (SNP and indel) mining from next generation sequences of diploid and polyploid genotypes.','A highly automated and configurable pipeline package will be delivered to the public and will be potentially useful in any self-pollinating species. This tool is needed immediately to handle the data expected from this project and to systematically format the variant discovery for the project database.\r',9),(25,'Genome-wide survey sequencing and bisulfite sequence','Survey (skim) sequences with and without sodium bisulfite treatment from a set of AAFC synthetic wheat lines selected for their differential vernalization requirements. Alignment of sequence data from selected germplasm including two diploid D genome parents, two tetraploid AABB genome parents and four synthetic lines, and prediction of DNA methylation sites. (Objective is conditional on availability of regulatory array from CTAG2)','The aligned survey sequence will provide a foundational genomic resource that can be used for marker development and to interpret DNA methylation patterns in these genotypes. Uncovering the source of novel phenotypic variation generated in genetically uniform lines that possess different vernalization requirements provides a unique opportunity to add to the knowledge of flowering time control. Generation of DNA methylation profiles of key AAFC germplasm will uncover polymorphisms associated with differing vernalization responses. The majority of wheat varieties grown in Canada are spring types adapted to the environment found on the Canadian prairies. Strategies to expand the variation available to wheat breeders often involve transferring favourable alleles from wild relatives through interspecific hybridization. The transfer of undesirable vernalization alleles often complicates these strategies. Information from this work may provide alternate strategies to eliminate vernalization d',10),(40,'RNA sequencing','An RNASeq analysis from the eight wheat lines characterized in previous objectives of meristem tissues pre- and post-vernalization will link to the small RNA objective. In addition, a global transcriptomic approach will be applied to obtain expression data from a wide range of tissues in all eight genotypes with specific emphasis on reproductive tissues. ','Here, we clearly see that all dataset layers are interlinked to provide a comprehensive picture of the mechanisms that could lead to differential gene expression and silencing. While it is impossible to look at a large number of traits, we have carefully selected the germplasm to investigate the winter/spring habit phenotype because it can be easily and clearly measured. Such phenotype will serve as proof of concept. However, phenotypic data of agronomic performance, fusarium head blight and leaf rust resistance of this germplasm is currently being collected through an on-going project. Eventually, we plan to apply the layered data to the other phenotypic data to better understand their genetic and epigenetic controls. Also special emphasis will be placed on reproductive tissues to provide insights into mechanisms of transgenerational inheritance in synthetics.\r',13),(50,'Backcross population development','Backcross populations (BC1F1 and BC2F1) will be created by crossing all four primary synthetics to an adapted elite spring wheat (Carberry) and winter wheat (Emerson).','Tissues from the populations will be sampled for future DNA and RNA experiments such as those described above to further elucidate gene silencing and activation mechanisms. The phenotypic data on the growth habit will provide useful information on such mechanisms for this trait.\r',15),(70,'DNA markers','Development and uptake of DNA markers in wheat, oat, and barley cultivar development programs through the direct involvement and engagement of wheat, barley, and oat breeders.','The bioinformatics analyses and tools will be validated in some real-world projects within the timeframe of this project, but the database and tools will continue to have high impact in breeding programs well-beyond project completion.\r',19),(5,'Germplasm','A definitive core collection of cultivated and wild germplasm in each of wheat, oat, and barley suitable for sequencing, allele mining, and SNP development.','This core germplasm collection will encompass a wide spectrum from wild relatives of the secondary and tertiary gene pools to progenitors of the primary gene pools (including both wild and domesticated forms), to synthetics, landraces and cultivars.  These two resources (DNA and seeds) will be made available for distribution to other Scientists for additional genotyping (e.g. genome sequencing, epigenetics, RNA-Seq) and/or phenotyping (disease, agronomics, quality) characterization. ',20),(10,'Oat gene capture array','The first global gene capture assay for oat.','The first novel gene capture array for use in oat will be published and shared openly with investigators in the oat community. Widespread uptake of this exome capture platform will benefit the community through the incremental development of experimental data and annotations, and the development of more diversity data by the community. ',21),(15,'Exome sequences','Deep coverage of exome sequences, assembled into accurate representations of gene space from core germplasm collections in wild and cultivated germplasm from wheat, oat, and barley, as well as synthetic hexaploids in wheat.','Ability to mine SNPs from cultivated germplasm, and gene variants from crop wild relatives. This sequence data will also provide information on the allelic diversity and underlying heterozygosity within progenitors, synthetics and synthetic-derived germplasm. In wheat, these data will be interpreted together with epigenetic analyses to identify mechanisms leading to gene silencing that balance the expression of homologous genes situated on the A, B and D genomes.  ',22),(30,'Histone modification analysis','Measurement of four chromatin marks throughout the wheat genome of the eight wheat lines characterized in previous objectives. These chromatin marks are known to activate or silence gene expression and their role in determining the observed flowering time variation in otherwise genetically uniform lines will be investigated.','Detection of specific histone methylation/acetylation profiles in key AAFC germplasm will uncover polymorphisms associated with different vernalization responses. Flowering time is among the most important traits under selection in breeding programs.  Understanding the regulation of this trait is beneficial to the wheat breeding industry.  The novel flowering time responses observed in genetically uniform lines generated through re-synthesis offers a unique opportunity to gain insight into how new combinations of AABB and DD genomes interact to generate novel phenotypic variation. ',23),(35,'Small RNA sequencing','Sequenced small RNA (siRNA) libraries from anthers, ovaries and meristems pre- and post-vernalization of the eight wheat lines characterized in previous objectives. Annotation and expression analyses of small RNA libraries.','The siRNAs originate mostly from repetitive elements, serving as silencing mechanisms. Those originating from well characterized families will be investigated. Mining of these data will allow us to understand the role these small non-coding molecules play in transgenerational inheritance of some epigenetic mechanisms such as the silencing or activation of repetitive elements and genes, phenomena that can create novel variation and result in phenotype differences. Bursts of transposition events in wheat have been reported for the Sasanda retroelement at a time coincident to the advent of hexaploid wheat. This led us to hypothesize that polyploidization could have caused a genomic stress that activated transposition. Here, we will test this hypothesis using newly created synthetic wheats that mimic the original hybridization events that occurred approximately ten thousand years ago. Understanding of these mechanisms could lead to the development of methods to “control” transposition for ',24),(45,'Candidate gene analysis','An in-depth sequence and diversity analysis in cultivated and wild germplasm of candidate genes in wheat, oat, and barley that show homology to genes implicated in fungal resistance, flowering time, cellulose synthesis, and hull adhesion. Prediction of synonymous vs non-synonymous mutations in coding regions, and assessment of relative divergence rate among different genes and gene families.','This work will provide information on new alleles for genes that are not present in primary gene pools of each species, and recommendations for further work and validation experiments using diverse alleles from target traits.  Furthermore, the analysis of model genes in collaboration with colleagues will facilitate the development of additional expertise and bioinformatics tools in our laboratories and those of collaborators. ',25),(55,'phylogeny analysis','Phylogeny analysis using defined gene sets at a level of species-diversity and gene-coverage never-before achieved will be conducted in oat. ','This work will provide a unique opportunity to refine and validate the relationships among species in the crop wild relatives of oat.  This information will be used to make educated decisions regarding the implementation of wide crosses in this crop. ',26),(60,'Database','A set of three crop genome databases for wheat, oat, and barley, developed and maintained in parallel, containing annotated gene predictions with SNP and Indel predictions from cultivated species, gene variant alignments from crop wild relatives, and annotated tracks identifying epigenetic variants in experimental material (see separate objectives). Databases will have a user-friendly query system allowing extraction of key data, or export of specific file formats (e.g. for SNP development or for phylogeny analysis).  Database will use elements of the open source GBrowse system as well as custom elements developed by Dr You’s team.','The database will be available initially to all collaborators, and by the end of the project, it will be published online and/or made available as an installable package. One type of user will be scientists performing experiments related to gene discovery or allele mining, who will then have pre-emptive data on gene variants, providing direct information for validation experiments, or the ability to better target new research hypotheses. The second type of user will be plant breeders, who currently lack the ability to quickly develop alternate or improved assays for targets of marker-assisted selection. ',27),(65,'Epigenetic database layers in wheat','Additional database layers (tracks) in the wheat genome database providing integrative views for the combined interpretation of genic and epigenomic data.','The individual processed and analyzed epigenetic datasets will have value on their own to answer specific hypotheses, but the real value will be the incorporation of all the data layers into a single database. Here, one will be able to understand the relationship between various data layers. For example, is this synthetic wheat a spring type because one of the vernalization genes is silenced through methylation of its promoter or through miRNA down-regulation or as a consequence of chromatin remodeling or a combination of each layer of regulation?  The analysis of these layers of gene regulation will uncover how quantitative trait variations manifest themselves.  Armed with this information, we will be able to ask why crosses between spring type durum cv. Langdon generate winter synthetics while crosses with the same diploid but other tetraploids do not. The measurable of this objective will be in the query power, ease of use, visualization and extraction tools. We anticipate that the ',28);
/*!40000 ALTER TABLE `objectives` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `participants`
--

DROP TABLE IF EXISTS `participants`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `participants` (
  `site` varchar(100) DEFAULT NULL,
  `name_id` varchar(1000) DEFAULT NULL,
  `cost_centre` varchar(100) DEFAULT NULL,
  `from_participant` varchar(1) DEFAULT NULL,
  `name1` varchar(100) DEFAULT NULL,
  `name2` varchar(100) DEFAULT NULL,
  `pi` varchar(1) DEFAULT NULL,
  `role` varchar(100) DEFAULT NULL,
  `responsibility` varchar(1000) DEFAULT NULL,
  `initials` varchar(10) DEFAULT NULL,
  `publishing_initials` varchar(10) DEFAULT NULL,
  `email` varchar(100) DEFAULT NULL,
  `contribution_percent` int(11) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=116 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `participants`
--

LOCK TABLES `participants` WRITE;
/*!40000 ALTER TABLE `participants` DISABLE KEYS */;
INSERT INTO `participants` VALUES ('Morden','You, Frank','5721BF','N','Frank','You','Y','Participant/Co-Lead','\"','Plant bioi','FY','F.M.',0,30),('Brandon','Badea, Ana','(A07732)','N','Ana','Badea','N','Participant','Barley breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','AB','A.','Ana.badea@agr.gc.ca',5,64),('Ottawa','Bekele, Wubishet','','N','Wubishet','Bekele','N','PRP','Reports to N. Tinker; Dr. Bekele brings expertise in biostatistics, bioinformatics, genomics, molecular breeding, and genetic diversity analysis in plants.  He will collaborate on development of the oat exome capture and on the analysis of resulting data.  Dr Bekele has experience in working with large project teams and will assist Dr Tinker in overall project management.','','','Wubishet.Bekele@agr.gc.ca',50,65),('Saskatoon','Bender, Tricia','','N','Tricia','Bender','N','Support','Reports to S. Robinson (EG-05) as indeterminate core technician.  Focus on work related to characterisation of epigenetic variation resulting from the generation of synthetic wheat lines.  Tricia will preform the immunoprecipitations, generate and help analyse the data','','','',50,66),('Morden','Bezte, Leslie','','N','Leslie','Bezte','N','Support','Reports to C. McCartney. Interface testing, interface utilization, and marker design and validation. Exome capture.','','','',10,67),('Morden','Brown, Susan','','N','Susan','Brown','N','Support','Reports to M. Jordan. Wheat markers identified during the project will be tested on genetic populations to identify markers linked to specific traits of interest.','','','',20,68),('Brandon','Burt, Andrew','5731BB','N','Andrew','Burt','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','AJB','A.J.','Andrew.Burt@agr.gc.ca',5,69),('Ottawa','Cloutier, Sylvie','5621CE or 5621EG','N','Sylvie','Cloutier','Y','Participant','Molecular geneticist. Responsible for exome capture of wheat wild relatives, synthetics and their parents, small RNA sequencing and RNA-Seq. She will work closely with Frank You to process the high throughput sequencing data and to conceptualise the database.','SC','S.','Sylvie.J.Cloutier@agr.gc.ca',20,70),('Swift Current','Colenutt, Theresa','','N','Theresa','Colenutt','N','Support','Reports to Knox/Cuthbert/Ruan. Test database application by analysis of wheat breeding material at Swift Current.','','','',5,71),('Ottawa','Couroux, Philippe','','N','Philippe','Couroux','N','Support','Reports to N. Tinker (CS2). Responsible for maintaining bioinformatics infrastructure at Ottawa RDC and assisting with high-throughput pipeline analysis of DNA sequence.','','','',10,72),('Swift Current','Cuthbert, Richard','5751AU','Y','Richard','Cuthbert','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','RC','R.D.','Richard.Cuthbert@agr.gc.ca',5,73),('Saskatoon','Diederichsen, Axel','5741AK','Y','Axel','Diederichsen','N','Participant','Curator of Plant Gene Resources Canada (PGRC).  Dr. Diederichsen will provide advice and data on germplasm selected for exome capture analysis, and will populate the PGRC database with new relevant information from the project','AD','A.','axel.diederichsen@agr.gc.ca',5,74),('Ottawa','Edwards, Tara','','N','Tara','Edwards','N','Support','Reports to S. Cloutier. Responsible to grow and care for the wheat wild relative germplasm collection, synthetics and their parental lines. She will sample tissues and extract high quality total RNA from a variety of germplasm, tissues and conditions and high quality genomic DNA from germplasm collection.','','','',20,75),('Morden','Enns, Suzanne','','N','Suzanne','Enns','N','Support','Reports to C. McCartney. Interface testing, interface utilization, and marker design and validation for wheat at Morden.','','','',10,76),('Ottawa','Fedak, George','5621CN','N','George','Fedak','N','Participant','Cereal cytogeneticist, provides expertise in crop wild relatives (wheat, oat, and barley), wide crosses, and introgressions. Evaluates new material','GF','G.','George.Fedak@agr.gc.ca',5,77),('Saskatoon','Fu, Yong-Bi','5741AL','Y','Yong-Bi','Fu','Y','Participant','plant population genetics and crop genetic diversity. Will perform and analyse exome capture sequence analysis in wild and cultivated barley.','YBF','Y.B.','Yong-Bi.Fu@agr.gc.ca',70,78),('Lethbridge','Graf, Robert','5761AV','N','Robert','Graf','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','RG','R.J.','Robert.Graf@agr.gc.ca',5,79),('Morden','Hiebert, Colin','5721BC','Y','Colin','Hiebert','Y','Participant','Wheat geneticist. Exome capture of wheat lines. Will evaluate suitability of SNP assays in his genetic research program. Will guide development of the web interface for the SNP database.','CH','C.W.','Colin.Hiebert@agr.gc.ca',10,80),('Saskatoon','Horbach, Carolee','','N','Carolee','Horbach','N','Support','Reports to Y-B Fu. Lab support for barley exome capture research at Saskatoon.','','','',50,81),('Saskatoon','Horner, Kyla','','N','Kyla','Horner','N','Support','Reports to S. Robinson (EG-03) as indeterminate project technician.  Focus on work related to characterisation of epigenetic variation resulting from the generation of synthetic wheat lines. Kyla will generate the DNA-seq, WGBS data.','','','',50,82),('Ottawa','Humphreys, Gavin','562127','N','Gavin','Humphreys','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','GH','D.G.','Gavin.Humphreys@agr.gc.ca',5,83),('Brandon','Jatinder, Sangha','','N','Sangha','Jatinder','N','Support','Reports to A. Badea. Provides barley germplasm preparation and lab analysis at Brandon.','','','',15,84),('Morden','Jordan, Mark','572143','N','Mark','Jordan','Y','Participant','Wheat geneticist. Exome capture of wheat lines. Will evaluate suitability of SNP assays in his genetic research program. Will guide development of the web interface for the SNP database.','MJ','M.C.','Mark.Jordan@agr.gc.ca',5,85),('Ottawa','Khanal, Raja','','N','Raja','Khanal','N','Collaborator','Barley breeder. Will evaluate the suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','RK','R.','raja.khanal@canada.ca',3,86),('Swift Current','Knox, Ron','575133','Y','Ron','Knox','N','Participant','Wheat geneticist. Will evaluate suitability of SNP assays in Swift Current breeding programs.  Will guide development of the web interface for the SNP database.','REK','R.E.','Ron.Knox@AGR.GC.CA',5,87),('Brandon','Kumar, Santosh','5731BA','N','Santosh','Kumar','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','SK','S.','Santosh.Kumar@agr.gc.ca',5,88),('Ottawa','Langille, Linda','','N','Linda','Langille','N','Support','Reports to G. Humphreys. Ms. Langille will be responsible for growing wheat plant material for study, DNA extractions, PCR reactions and SSR or SNP data collection (i.e. reading electropherograms or fluorometer plate reads) and collation.','','','',10,89),('Lethbridge','Larsen, Jamie','5761CE','Y','Jamie','Larsen','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','JL','J.R.','Jamie.Larsen@agr.gc.ca',5,90),('Morden','Mardli, Ghassan','','N','Ghassan','Mardli','N','Support','Reports to C. Hiebert.  Screening SNP markers with parental lines from mapping populations, genotyping mapping populations with appropriate SNP markers, collecting and entering genotyping data, testing key SNP markers on a panel of wheat cultivars to identify the most diagnostic markers for targeted traits.','','','',5,91),('Morden','McCartney, Curt','572173','N','urt','McCartney','Y','Participant','Wheat and oat geneticist. Exome capture of wheat lines. Will evaluate suitability of SNP assays in his genetic research program. Will guide development of the web interface for the SNP database.','CM','C.A.','Curt.McCartney@agr.gc.ca',10,92),('Swift Current','Meyer, Brad','','N','Brad','Meyer','N','Support','Reports to Knox/Cuthbert/Ruan. Interface testing, interface utilization and wheat marker design and validation at Swift Current.','','','',5,93),('Brandon','Mitchell Fetch, Jennifer','5721AR','N','Jennifer','Mitchell Fetch','N','Participant','Oat breeder. Will evaluate the suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','JMF','J.','Jennifer.MitchellFetch@AGR.GC.CA',5,94),('Brandon','Nakhforoosh, Alireza','','N',' Alireza','Nakhforoosh','N','PRP','Oat - with Jennifer','','','',0,95),('Brandon','Pandurangan, Sudhakar','','N','Sudhakar','Pandurangan','N','Support','Reports to A. Burt and S. Kumar. Applying SNP markers to screen wheat breeding populations at Brandon.','','','',5,96),('Saskatoon','Peterson, Gregory','','N','Gregory','Peterson','N','Support','Reports to Y-B Fu. Provides bioinformatics support for barley exome capture research at Saskatoon.','','','',50,97),('Swift Current','Piche, Isabelle','','N','Isabelle','Piche','N','Support','Reports to Knox/Cuthbert/Ruan. Marker enhancement oversight for interface testing and application to wheat breeding at Swift Current.','','','',5,98),('Morden','Popovic, Mira','','N','Mira','Popovic','N','Support','Reports to C. Hiebert.  Screening SNP markers with parental lines from mapping populations, genotyping mapping populations with appropriate SNP markers, collecting and entering genotyping data, testing key SNP markers on a panel of wheat cultivars to identify the most diagnostic markers for targeted traits.','','','',5,99),('Morden','PRP-TBA','','N','PRP','TBA','N','PRP','Postdoctoral fellow reporting to Dr. You in Morden with expertise in bioinformatics and computational genomics, to be hired under the AAFRC PRP program.  The fellow will assist with developing the database, pipelines, and user interface, and will co-supervise co-op students in analysing data and populating the database.','','','',100,100),('Lethbridge','Randhawa, Harpinder','5761AC','N','Harpinder','Randhawa','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','HR','H.S.','Harpinder.Randhawa@agr.gc.ca',5,101),('Ottawa','Ravichandran, Sridhar','','N','Sridhar','Ravichandran','N','PRP','Postdoctoral fellow reporting to S. Cloutier in Ottawa with expertise in genome and epigenome analysis, to be hired under the AAFRC PRP program.  The fellow will assist with the wheat exome capture activity and the wheat epigenetic exome, miRNA and RNA-Seq activities (approximately 50% on exome activity and 50% on epigenomics).','','','',100,102),('Saskatoon','Robinson, Steve','5741BX','N','Steve','Robinson','Y','Participant','Expertise in plant genetics, epigenetics, epigenomics and bioinformatics.  As an AAFC Participant, Dr. Robinson will contribute to characterising the role of epigenetic variation between the parental and sythtetic wheat lines varying in flowering time growth habit. He is also involved in the design and development of the wheat regulatory region capture array as part of the CTAG2 project.','SR','S.J.','Steve.Robinson@agr.gc.ca',30,103),('Swift Current','Ruan, Yuefeng','5751AX','Y','Yuefeng','Ruan','N','Participant','Wheat breeder. Will evaluate suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','YR','Y.','Yuefeng.Ruan@agr.gc.ca',5,104),('Ottawa','Student1-TBA','','N','Student','TBA','N','Student','Coop students at Ottawa with G. Fedak (2 terms per year, possibly same or different students) to assist with population development in wheat synthetic backcrosses','','','',100,105),('Saskatoon','Student2-TBA','','N','Student','TBA','N','Student','Coop student at Saskatoon with Y-B. Fu (1 term per year, possibly same or different students) to assist with plant growth, tissue sampling, phenotypic observation, and maintenance of germplasm at PGRC','','','',100,106),('Saskatoon','Student3-TBA','','N','Student','TBA','N','Student','Coop student at Saskatoon with S. Robinson (1 term per year, possibly same or different students) to assist with plant growth, tissue sampling, and laboratory analysis of genomic and epigenomic samples.','','','',100,107),('Morden','Student4-TBA','','N','Student','TBA','N','Student','Coop students at Morden with Frank You (2 terms per year, possibly same or different students) to assist database and interface development.','','','',100,108),('Morden','Student5-TBA','','N','Student','TBA','N','Student','Coop students at Morden with Frank You (2 terms per year, possibly same or different students) to assist with SNP data analysis and populating database with new results.','','','',100,109),('Ottawa','Sunstrum, Fred','','N',' Fred','Sunstrum','N','Student','Coop student at Ottawa with N. Tinker  (1 terms in year 1) to assist with plant growth, tissue sampling, phenotypic observation (year1)','','','',100,110),('Ottawa','Tinker, Nick','5621CV','Y','Nicholas','Tinker','Y','Participant/Lead','Plant genomics and bioinformatics / allele mining in wild oat / molecular breeding applications in oat.  Will design the oat exome capture assays, perform sequence analysis, and advise on design of database.','NT','N.A.','Nick.Tinker@agr.gc.ca',50,111),('Ottawa','Wight, Charlene','','N','Charlene','Wight','N','Support','Reports to N. Tinker (EG5). Preparing oat DNA samples, growing plant material, preparing exon capture libraries, managing the oat molecular laboratory.','','','charlene.wight@agr.gc.ca',40,112),('Ottawa','Yan, Weikai','','N','Weikai','Yan','N','Collaborator','Oat breeder. Will evaluate the suitability of SNP assays in their breeding program.  Will guide development of the web interface for the SNP database.','WY','W.','Weikai.Yan@agr.gc.ca',3,113),('Morden','Yao, Zhen','','N','Zhen','Yao','N','Support','Reports to Frank You.  Responsible for data integration and database development at Morden.','','','',30,114),('Brandon','Zhang, Shunzhen','','N','Shunzhen','Zhang','N','Support','Reports to A. Burt and S. Kumar. Applying SNP markers to screen wheat breeding populations at Brandon.','','','',5,115);
/*!40000 ALTER TABLE `participants` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `publications`
--

DROP TABLE IF EXISTS `publications`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `publications` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `year` varchar(100) DEFAULT NULL,
  `title` varchar(1000) DEFAULT NULL,
  `citation` varchar(1000) DEFAULT NULL,
  `link` varchar(1000) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `publications`
--

LOCK TABLES `publications` WRITE;
/*!40000 ALTER TABLE `publications` DISABLE KEYS */;
INSERT INTO `publications` VALUES (1,'2017','Genomic tool for fine mapping and marker development of wheat resistance genes','You FM, Li P, Xiao J, Kassa MT, McCartney, Hiebert CW, Jordan MC, Cloutier S, Poznial C, Fobert P, Wang X, Luo MC and Dvroak J. Genomic tool for fine mapping and marker development of wheat resistance genes. Proceedings of 13th International Wheat Genetics Symposium, April 23-28, 2017, Tulln, Austria, Page 214.','');
/*!40000 ALTER TABLE `publications` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `support`
--

DROP TABLE IF EXISTS `support`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `support` (
  `name_id` varchar(1000) DEFAULT NULL,
  `name1` varchar(100) DEFAULT NULL,
  `name2` varchar(100) DEFAULT NULL,
  `site` varchar(100) DEFAULT NULL,
  `team` varchar(100) DEFAULT NULL,
  `role` varchar(100) DEFAULT NULL,
  `responsibility` varchar(1000) DEFAULT NULL,
  `contribution_percent` int(11) DEFAULT NULL,
  `email` varchar(100) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=30 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `support`
--

LOCK TABLES `support` WRITE;
/*!40000 ALTER TABLE `support` DISABLE KEYS */;
INSERT INTO `support` VALUES ('Bekele, Wubishet','Wubishet','Bekele','Ottawa','Tinker','PRP','Reports to N. Tinker; Dr. Bekele brings expertise in biostatistics, bioinformatics, genomics, molecular breeding, and genetic diversity analysis in plants.  He will collaborate on development of the oat exome capture and on the analysis of resulting data.  Dr Bekele has experience in working with large project teams and will assist Dr Tinker in overall project management.',50,'Wubishet.Bekele@agr.gc.ca\r',1),('Bender, Tricia','Tricia','Bender','Saskatoon','Robinson','Support','Reports to S. Robinson (EG-05) as indeterminate core technician.  Focus on work related to characterisation of epigenetic variation resulting from the generation of synthetic wheat lines.  Tricia will preform the immunoprecipitations, generate and help analyse the data',50,'\r',2),('Bezte, Leslie','','','Morden','McCartney','Support','Reports to C. McCartney. Interface testing, interface utilization, and marker design and validation. Exome capture.',10,'\r',3),('Brown, Susan','','','Morden','Jordan','Support','Reports to M. Jordan. Wheat markers identified during the project will be tested on genetic populations to identify markers linked to specific traits of interest.',20,'\r',4),('Colenutt, Theresa','','','Swift Current','Knox & Cuthbert & Ruan','Support','Reports to Knox/Cuthbert/Ruan. Test database application by analysis of wheat breeding material at Swift Current.',5,'\r',5),('Couroux, Philippe','','','Ottawa','Tinker','Support','Reports to N. Tinker (CS2). Responsible for maintaining bioinformatics infrastructure at Ottawa RDC and assisting with high-throughput pipeline analysis of DNA sequence.',10,'\r',6),('Edwards, Tara','Tara','Edwards','Ottawa','Cloutier','Support','Reports to S. Cloutier. Responsible to grow and care for the wheat wild relative germplasm collection, synthetics and their parental lines. She will sample tissues and extract high quality total RNA from a variety of germplasm, tissues and conditions and high quality genomic DNA from germplasm collection.',20,'\r',7),('Enns, Suzanne','','','Morden','McCartney','Support','Reports to C. McCartney. Interface testing, interface utilization, and marker design and validation for wheat at Morden.',10,'\r',8),('Horbach, Carolee','','','Saskatoon','Fu','Support','Reports to Y-B Fu. Lab support for barley exome capture research at Saskatoon.',50,'\r',9),('Horner, Kyla','Kyla',' Horner','Saskatoon','Robinson','Support','Reports to S. Robinson (EG-03) as indeterminate project technician.  Focus on work related to characterisation of epigenetic variation resulting from the generation of synthetic wheat lines. Kyla will generate the DNA-seq, WGBS data.',50,'\r',10),('Jatinder, Sangha','','','Brandon','Badea','Support','Reports to A. Badea. Provides barley germplasm preparation and lab analysis at Brandon.',15,'\r',11),('Langille, Linda','','','Ottawa','Humphreys','Support','Reports to G. Humphreys. Ms. Langille will be responsible for growing wheat plant material for study, DNA extractions, PCR reactions and SSR or SNP data collection (i.e. reading electropherograms or fluorometer plate reads) and collation.',10,'\r',12),('Mardli, Ghassan','Ghassan','Mardli','Morden','Hiebert','Support','Reports to C. Hiebert.  Screening SNP markers with parental lines from mapping populations, genotyping mapping populations with appropriate SNP markers, collecting and entering genotyping data, testing key SNP markers on a panel of wheat cultivars to identify the most diagnostic markers for targeted traits.',5,'\r',13),('Meyer, Brad','','','Swift Current','Knox & Cuthbert & Ruan','Support','Reports to Knox/Cuthbert/Ruan. Interface testing, interface utilization and wheat marker design and validation at Swift Current.',5,'\r',14),('Pandurangan, Sudhakar','','','Brandon','Burt & Kumar','Support','Reports to A. Burt and S. Kumar. Applying SNP markers to screen wheat breeding populations at Brandon.',5,'\r',15),('Peterson, Gregory','','','Saskatoon','Fu','Support','Reports to Y-B Fu. Provides bioinformatics support for barley exome capture research at Saskatoon.',50,'\r',16),('Piche, Isabelle','','','Swift Current','Knox & Cuthbert & Ruan','Support','Reports to Knox/Cuthbert/Ruan. Marker enhancement oversight for interface testing and application to wheat breeding at Swift Current.',5,'\r',17),('Popovic, Mira',' Mira','Popovic','Morden','Hiebert','Support','Reports to C. Hiebert.  Screening SNP markers with parental lines from mapping populations, genotyping mapping populations with appropriate SNP markers, collecting and entering genotyping data, testing key SNP markers on a panel of wheat cultivars to identify the most diagnostic markers for targeted traits.',5,'\r',18),('PRP-TBA','','','Morden','You','PRP','Postdoctoral fellow reporting to Dr. You in Morden with expertise in bioinformatics and computational genomics, to be hired under the AAFRC PRP program.  The fellow will assist with developing the database, pipelines, and user interface, and will co-supervise co-op students in analysing data and populating the database.',100,'\r',19),('Ravichandran, Sridhar','Sridhar','Ravichandran','Ottawa','Cloutier','PRP','Postdoctoral fellow reporting to S. Cloutier in Ottawa with expertise in genome and epigenome analysis, to be hired under the AAFRC PRP program.  The fellow will assist with the wheat exome capture activity and the wheat epigenetic exome, miRNA and RNA-Seq activities (approximately 50% on exome activity and 50% on epigenomics).',100,'\r',20),('Student1-TBA','','','Ottawa','Fedak','Student','Coop students at Ottawa with G. Fedak (2 terms per year, possibly same or different students) to assist with population development in wheat synthetic backcrosses',100,'\r',21),('Student2-TBA','','','Saskatoon','Fu','Student','Coop student at Saskatoon with Y-B. Fu (1 term per year, possibly same or different students) to assist with plant growth, tissue sampling, phenotypic observation, and maintenance of germplasm at PGRC',100,'\r',22),('Student3-TBA','','','Saskatoon','Robinson','Student','Coop student at Saskatoon with S. Robinson (1 term per year, possibly same or different students) to assist with plant growth, tissue sampling, and laboratory analysis of genomic and epigenomic samples.',100,'\r',23),('Student4-TBA','','','Morden','You','Student','Coop students at Morden with Frank You (2 terms per year, possibly same or different students) to assist database and interface development.',100,'\r',24),('Student5-TBA','','','Morden','You','Student','Coop students at Morden with Frank You (2 terms per year, possibly same or different students) to assist with SNP data analysis and populating database with new results.',100,'\r',25),('Sunstrum, Fred','','','Ottawa','Tinker','Student','Coop student at Ottawa with N. Tinker  (1 terms in year 1) to assist with plant growth, tissue sampling, phenotypic observation (year1)',100,'\r',26),('Wight, Charlene','Charlene','Wight','Ottawa','Tinker','Support','Reports to N. Tinker (EG5). Preparing oat DNA samples, growing plant material, preparing exon capture libraries, managing the oat molecular laboratory.',40,'charlene.wight@agr.gc.ca\r',27),('Yao, Zhen','Zhen','Yao','Morden','You','Support','Reports to Frank You.  Responsible for data integration and database development at Morden.',30,'\r',28),('Zhang, Shunzhen','','','Brandon','Burt & Kumar','Support','Reports to A. Burt and S. Kumar. Applying SNP markers to screen wheat breeding populations at Brandon.',5,'\r',29);
/*!40000 ALTER TABLE `support` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `tools`
--

DROP TABLE IF EXISTS `tools`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `tools` (
  `title` varchar(100) DEFAULT NULL,
  `category` varchar(100) DEFAULT NULL,
  `description` varchar(1000) DEFAULT NULL,
  `link` varchar(1000) DEFAULT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `tools`
--

LOCK TABLES `tools` WRITE;
/*!40000 ALTER TABLE `tools` DISABLE KEYS */;
INSERT INTO `tools` VALUES ('ConservedPrimers ','High Throughput Marker/Primer Development ','A high-throughput pipeline for intron-flanking PCR primer design for SNP discovery or variation dection',NULL,3),('FPCBrowser ','Physical Mapping and Fingerprint Data Analysis ','A Java-written, platform-dependant and GUI-based software tool. FPCBrowser was initially designed as a portable physical map viewer to comprehensively view FPC contig maps and related information such as clones, fingerprints, and markers in multiple platforms. A Java-based relational database, HSQLDB, was adopted in FPCBrowser to store all source data of a FPC contig map and the fingerprints of BAC clones in a physical mapping project. Some utilities have been implemented to extract useful information from FPC databases. A program module of the clone deconvolution was appended to FPCBrowser for BAC library screening using the new high-throughput five-dimensional clone pooling strategy.',NULL,4),('GenoProfiler ','Physical Mapping and Fingerprint Data Analysis ','GenoProfiler is a cross-platform software package for the fully automated processing of fingerprinting data from ABI 3100/3700/3730 Genetic Analyzer. This software provides the algorithms to accurately extract the traces and peak data (sizes, peak height, peak area, and scan number or data point) directly from sample files obtained from ABI 3100/3700 Genetic Analyzer without using ABI GeneScan, GeneMapper, or other software, and automatically remove the background (noise) bands and \"false\" bands, and create edited size files with FPC size format which can be directly used for contig assembly using FPC software.',NULL,5),('HeatMap ','Genome Analysis and SNP Discovery ','A heat map drawing tool based on distribution of any feature on genome/chromosomes.',NULL,6),('MAD Analysis Pipeline Package ','Statistical Analysis of Experimental Data ','A software tool to analyze experimental data of the type 2 modified augmented design (MAD). The package combing SAS software and perl scripts supports automated analyses for single MAD experiments and joint analyses for multiple experiments (years and locations).',NULL,7),('RJPrimers ','High Throughput Marker/Primer Development ','RJPrimers is a high-throughput software tool to identify unique repeat junction and design TE-based primers for high-throughput marker development. This tool first identifies potentially unique repeat junctions using BLAST against fully annotated repeat databases and a repeat junction finding algorithm, and then designs TE based primers using Primer3 and BatchPrimer3. Five primer design strategies of TE based PCR markers have been implemented in this tool, including RJM, RJJM, ISBP, RBIP, and IRAP. Both a web-based server and a command line based pipeline have been implemented to meet different requirements. RJPrimers takes sequences in FASTA format as input and generates several pages of primer design results, including an HTML table page and a tab-delimited text file listing all designed primers and primer properties, and a detailed primer view page for each sequence with successfully designed primers.',NULL,8),('SeqUtils ','Physical Mapping and Fingerprint Data Analysis ','A Java-written, platform-dependant and GUI-based software tool collections, including the Sequence retrieving utility, Gel Retrieving and Primer design etc.',NULL,9),('AGSNP','Genome Analysis and SNP Discovery','An annotation-based, genome-wide SNP discovery pipeline package using NGS data for large and complex genomes without a reference genome sequence. Roche 454 shotgun reads with low genome coverage of one individual are annotated in order to distinguish single-copy sequences and repeat junctions from repetitive sequences and sequences shared by paralogous genes. Multiple genome equivalents of shotgun reads of another individual generated with SOLiD or Solexa are then mapped to the annotated Roche 454 reads to identify putative SNPs. The pipeline is suitable for SNP discovery in genomic libraries of complex genomes and does not require a reference genome sequence. The pipeline is applicable to all current NGS platforms, provided that at least one of them generates relatively long reads. This pipeline package is available upon request.','',10),('BatchPrimer3','High Throughput Marker/Primer Development','BatchPrimer3 is a comprehensive web primer design program using Primer3 core program as a major primer design engine to design different types of PCR primers and sequencing primers in a high-through manner. BatchPrimer3 allows users to design several types of primers including generic primers, hybridization oligos, SSR primers together with SSR detection, and SNP genotyping primers (including single-base extension primers, allele-specific primers, and tetra-primers for tetra-primer ARMS PCR), as well as DNA sequencing primers. A batch input of large number of sequences and a tab-delimited result output greatly facilitates rapid primer design and ordering process.','',11);
/*!40000 ALTER TABLE `tools` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2018-02-19 13:48:07
